content/tutorials/manage-the-rippled-server/configuration/configure-statsd.md

```diff
@@ -0,0 +1,57 @@
+# Configure StatsD
+
+ The client implementation of [StatsD](https://github.com/statsd/statsd) within rippled can be enabled to export health and behavioral information about the server. Those metrics can be consumed and visualized through [`rippledmon`](https://github.com/ripple/rippledmon) or any other collector that accepts StatsD formatted metrics. 
+
+## Configuration Steps
+
+ To enable StatsD on your rippled server, perform the following steps:
+
+1. Configure rippledmon on a separate machine. For more information about configuring `rippledmon` see the `rippledmon` [repository](https://github.com/ripple/rippledmon).
+
+
+0. Add the `[insight]` stanaza to your `rippled`'s config file. The stanza should contain the address that you would like to send StatsD metrics to followed by the port number that `rippledmon` is listneing on.
```

Comment:
- Please explain the prefix. Should it be separate for each of your rippled instances, or should you group them, or should all of your instances use the same prefix?

- I suggest shortening the second sentence to the following:

    > For the `address`, use the IP address and port where `rippledmon` is listening.